tests/custom_steps.js

```diff
@@ -76,4 +77,55 @@ module.exports = () => actor({
       this.wait(interval);
     }
   },
+
+  /**
+   * Create URL method
+   *
+   * @param url start
+   * @param parameters object
+   * @returns {Promise<void>}
+   *
+   * @example
+   * getDashboardUrlWithParams('http://example.com', { environment: 'ps-dev', from: 'now-1' });
+   */
+  getDashboardUrlWithParams(url, parameters) {
```

Comment:
please rename it to `buildUrlWithParams`

Reply:
ok